Revisit manifests and controlled resources list in Istio operator (#664)
kyma-project/istio: docs/release-notes/1.5.0.md

## New Features

> - Istio installation version details is now specified only in the operator files [in here](https://github.com/kyma-project/istio/blob/main/internal/istiooperator) as single source of truth [#664](https://github.com/kyma-project/istio/pull/664).

Reviewer: ```suggestion
- The version details for Istio installation are now only specified in the files located in the [`istiooperator`](https://github.com/kyma-project/istio/blob/main/internal/istiooperator) directory [#664](https://github.com/kyma-project/istio/pull/664).
```
